Controller/PostController.php

```diff
@@ -23,9 +23,11 @@
 class PostController extends Controller
 {
     /**
+     * @param Request $request
+     *
      * @return RedirectResponse
      */
-    public function homeAction()
+    public function homeAction(Request $request = null)
```

Comment:
Yes, but you're right. If your override don't provide parameter, this will not work.